modules/videogrid-remote/pom.xml

```
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>opencast-videogrid-remote</artifactId>
+  <name>Opencast :: videogrid-remote</name>
+  <packaging>bundle</packaging>
+  <parent>
+    <groupId>org.opencastproject</groupId>
+    <artifactId>base</artifactId>
+    <version>8-SNAPSHOT</version>
```

```suggestion
    <version>9-SNAPSHOT</version>
```
Seems that you develop over 8.x, check if works with 9.x

---

That is a very, very good idea, I'll do that.